fairscale/nn/moe/top2gate.py

```diff
@@ -29,7 +29,6 @@ def gumbel_rsample(shape: Tuple, device: torch.device) -> Tensor:
 def top2gating(logits: torch.Tensor) -> Tuple[Tensor, Tensor, Tensor]:
     """Implements Top2Gating on logits."""
     gates = F.softmax(logits, dim=2)
-    min_logit = torch.finfo(logits.dtype).min  # type: ignore
 
     # gates has shape of GSE
     num_tokens = gates.shape[1]
```

Comment:
nits: perhaps this line and the line below can be combined into:

num_tokens, num_experts = gates.shape?